apps/emqx/src/emqx_cm.erl

```diff
@@ -96,6 +102,11 @@
 
 -define(T_TAKEOVER, 15000).
 
+%% linting overrides
+-elvis([ {elvis_style, invalid_dynamic_call, #{ignore => [emqx_cm]}}
```

Comment:
I tried to be more specific about the functions to be ignored, but apparently Elvis does not honor the `ignore`s that are not modules...